src/app/components/search/search-results/search-results.component.ts

```diff
@@ -154,6 +153,17 @@ export class SearchResultsComponent extends PageComponent implements OnInit {
     }
   }
 
+  onSearchError(error: { message: any }) {
+    const { statusCode } = JSON.parse(error.message).error;
+
+    const message =
+      this.translationService.instant(
+        `APP.BROWSE.SEARCH.ERRORS.${statusCode}`
```

Comment:
instant seems to return the translation key when the translation is not found.

Reply:
thanks for spotting, updated the code